app/src/main/java/org/mozilla/fenix/home/intent/StartSearchIntentProcessor.kt

```diff
@@ -23,38 +23,43 @@ class StartSearchIntentProcessor : HomeIntentProcessor {
 
     override fun process(intent: Intent, navController: NavController, out: Intent): Boolean {
         val event = intent.extras?.getString(HomeActivity.OPEN_TO_SEARCH)
+
         return if (event != null) {
-            val source = when (event) {
-                SEARCH_WIDGET -> {
-                    SearchWidget.newTabButton.record(NoExtras())
-                    MetricsUtils.Source.WIDGET
-                }
-                STATIC_SHORTCUT_NEW_TAB,
-                STATIC_SHORTCUT_NEW_PRIVATE_TAB,
-                PRIVATE_BROWSING_PINNED_SHORTCUT -> {
-                    MetricsUtils.Source.SHORTCUT
-                }
-                else -> null
+            return openSearch(event, navController, out)
```

Comment:
Comparing before and after it seems to me that `openSearch` contains the exact same code as before just extracted now in a new method?
If so this change probably won't be needed with preserving git history being more important.

Reply:
The structure of the code changed when I modified StartSearchIntentProcessor. I removed the changes but I left the modified structure. I'll revert back to the original code structure. Thank you for pointing that. I'll keep that in mind when I do future changes.